src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -15,11 +15,11 @@ import FastImage from 'react-native-fast-image';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
 import MemoIcArrowBackCircle from '../../../assets/arrow/Ic_arrow_back_circle';
 import ShareIconCircle from '../../../assets/icons/Ic_share_circle';
-import TopicDefaultIcon from '../../../assets/topic.png';
+import TopicSignedDefaultIcon from '../../../assets/dp-topics-signed.png';
+import TopicAnonDefaultIcon from '../../../assets/dp-topics-anon.png';
 import {Shimmer} from '../../../components/Shimmer/Shimmer';
 import dimen from '../../../utils/dimen';
 import {normalize} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated to include new icon assets and remove an unused hook. This change seems fine as long as the new icons are present in the specified path and the removed hook is not used anywhere else in the file.
```diff
- import TopicDefaultIcon from '../../../assets/topic.png';
+ import TopicSignedDefaultIcon from '../../../assets/dp-topics-signed.png';
+ import TopicAnonDefaultIcon from '../../../assets/dp-topics-anon.png';
- import useChatClientHook from '../../../utils/getstream/useChatClientHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->